tests/braille-specs/hu-hu-g1_braille_input_forward.yaml

```diff
@@ -178,3 +178,21 @@ tests:
   - ["1zzs", ⠼⠁⠜⠜]
   - ["1zzs1", ⠼⠁⠜⠜⠼⠁]
   - ["1azzs", ⠼⠁⠠⠁⠜⠜]
+#Following testcases testing redesigned old compatibility purpose translation rules,
+#with affects only back-translation
```

Comment:
Is this a cut and paste error? In other words is this comment correct here?

Reply:
I think the comment is right, perhaps grammar not. :-):-)
I wanted to indicate in the forward test that the scores under the comment should be displayed differently when translating in that direction than when translating backwards.
If you have a better suggestion based on the test cases under the comment, how to write a good comment for forward tests, I will be happy to improve the comment.

Reply:
Chris, what do you think if keeps only following comment part the tests/braille-specs/hu-hu-g1_braille_input_forward.yaml?
```
#Following testcases shows the right forward translation when the unicode simbols presents
#with Braille-displays
```

Following comment part are we simple remove from hu-hu-g1_braille_input_forward.yaml before the new testcases:
```
#Following testcases testing redesigned old compatibility purpose translation rules,
#with affects only back-translation
```

This style change you would like, or would like more simpler comment section before new tests?

Attila